### ISSUE TITLE
Suggestion: `fanReqResult`

### ISSUE THREAD

**Issue by marcotoniut** [imalsogreg/servant-reflex#75]
Here's a function that I see myself using constantly.

```hs
fanReqResult :: Reflex t => Event t (ReqResult tag a) -> (Event t Text, Event t a)
fanReqResult = fmapMaybe reqFailure &&& fmapMaybe reqSuccess

mainWidget = do
  (e_f, e_s) <- fanReqResult <$> fetch
  ...
```

Wouldn't it be considered a worthy addition to the library?

**Comment by imalsogreg** [comment]
I like it! Want to send a PR, or should I just copy this in?

**Comment by marcotoniut** [comment]
Neat! I'll make it into a PR later today, don't sweat it.

**Comment by imalsogreg** [comment]
Closed by #76
